AirCasting/SearchAndFollow/SearchLocation/SearchViewModel.swift

```diff
@@ -44,8 +44,8 @@ class SearchViewModel: ObservableObject {
         self.measurementTypes.first(where: { $0.name == param })?.isSelected = true
         switch measurementType(from: param) {
         case .particulateMatter: self.sensorTypes = [
-            .init(isSelected: false, name: SensorType.AB3and2.capitalizedName),
-            .init(isSelected: true, name: SensorType.OpenAQ.capitalizedName),
+            .init(isSelected: true, name: SensorType.AB3and2.capitalizedName),
+            .init(isSelected: false, name: SensorType.OpenAQ.capitalizedName),
             .init(isSelected: false, name: SensorType.PurpleAir.capitalizedName)
```

Comment:
If "openAQ and PurpleAir have revoked free access to their data via their APIs" then should we still have those? 🤔

Reply:
I was thinking about this and came up with some ideas. Disabling the view part, blocks possible interaction with user<>logic. I thought about having it in here for the future (not to reinvent the wheel and be able to easily revert those changes. I am making small improvement in terms of init right now, please tell me what you think

Reply:
Good point, let's leave it then :)